pages/common/ansible.md

```diff
@@ -27,3 +27,7 @@
 - Execute a command using a custom inventory file:
 
 `ansible {{group}} -i {{inventory_file}} -m command -a '{{my_command}}'`
+
+- List the groups in an inventory:
+
+`ansible localhost -m debug -a 'var=groups.keys()'`
```

Comment:
```suggestion
`ansible localhost -m debug -a '{{var=groups.keys()}}'`
```

Reply:
If the change suggestion from @bl-ue is done, then it's good to go.